docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+
+### Answering questions
+
+If you have a question and you can't find the answer in the documentation or issues, the next step is to submit a [question.](https://github.com/dell/<<repo>>/issues/new?labels=type%2Fquestion&template=ask-a-question.md&title=%5BQUESTION%5D%3A)
+
+We'd love your help in answering pending questions asked by other CSM users.
+
+## Triage issues
+
+Triage helps ensure that issues resolve quickly by:
+
+* Ensuring the issue's intent and purpose is conveyed precisely. This is necessary because it can be difficult for an issue to explain how an end user experiences a problem and what actions they took.
+* Giving a contributor the information they need before they commit to resolving an issue.
+* Lowering the issue count by preventing duplicate issues.
+* Streamlining the development process by preventing duplicate discussions.
+
+If you don't have the knowledge or time to code, consider helping with _issue triage_. The Dell dellemc.enterprise_sonic community will thank you for saving them time by spending some of yours.
+
+Read more about the ways you can [Triage issues](ISSUE_TRIAGE.md).
+
+## Testing
+
+See [here](https://github.com/dell/<<repo>>/tree/collections/ /README.md) for further information on testing.
+
+## Debugging
+To debug <product> using IDE, see [here](https://github.com/dell/<repo name>/tree/collections/docs/debug.md)
+
+
+## Your first contribution
+
+Unsure where to begin contributing? Start by browsing issues labeled `beginner friendly` or `help wanted`.
+
+* [Beginner-friendly](https://github.com/dell/<<repo>>/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) issues are generally straightforward to complete.
+* [Help wanted](https://github.com/dell/<<repo>>/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) issues are problems we would like the community to help us with regardless of complexity.
+
+When you're ready to contribute, it's time to create a pull request.
+
+## Branching
+
+* [Branching Strategy](BRANCHING.md) 
+
+## Signing your commits
+
+We require that developers sign off their commits to certify that they have permission to contribute the code in a pull request. This way of certifying is commonly known as the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). We encourage all contributors to read the DCO text before signing a commit and making contributions.
+
+GitHub will prevent a pull request from being merged if there are any unsigned commits.
+
+### Signing a commit
+
+GPG (GNU Privacy Guard) will be used to sign commits.  Follow the instructions [here](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits) to create a GPG key and configure your GitHub account to use that key.
+
+Make sure you have your user name and e-mail set.  This will be required for your signed commit to be properly verified.  Check the following references:
```

Comment:
```suggestion
Make sure you have your username and e-mail set.  This will be required for your signed commit to be properly verified.  Check the following references:
```

Reply:
Addressed as suggested

Reply:
The change has been incorporated.

Reply:
Verified.